conda/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set data= load_setup_py_data() %}
-{% set name= "ts-fiber-spectrograph" %}
+{% set name= ts-fiber-spectrograph %}
```

Comment:
Please just set the name directly to ts-fiber-spectrograph at line 4 below and skip this variable. Surely it is not being used anywhere else?